migrations/20210322094946-create-routine.js

```diff
@@ -1,7 +1,11 @@
 'use strict';
+
```

Comment:
Are you ignoring lint? there are empty spaces that should not be there

Reply:
no eslint is configured its a line break